analysis/staticAnalysisReport.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+title: "Eval Static Analysis"
+author: "Chakshu Goyal"
+date: "07/06/2018"
+output:
+  html_document: default
+  pdf_document: default
+---
+
+## 
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library(ggplot2)
+library(tidyverse)
+library(tools)
+```
+
+## Load the data
+Usage of the function eval in R-packages
+```{r}
+evaldf <- read_csv("evals.csv")
+```
+
+## Plot occurences of evals in packages
+```{r one}
+# Occurences of evals in each package plot
```

Comment:
Better than a comment is to actually write this in the Rmd file above the snippet as a header.
In this however, this comment is more less in the plot title.

Reply:
Yes. Acknowledged.